proxies/shared/resources/jsc/Messages.Create.Validate.js

```diff
@@ -1,8 +1,231 @@
 /**
- * This script validates the data in the body of a POST request to the /v1/messages endpoint, returning an
+ * This script validates the data in the body of a POST request to the /v1/message-batches endpoint, returning an
  * array of up to 100 error responses to be returned by Apigee as a 400 error response.
  *
- * It is called by the JavaScript.Messages.Create.Validate policy.
+ * It is called by the JavaScript.MessageBatches.Create.Validate policy.
  */
 
-context.setVariable("errors", null);
+// Regex to validate the id is a valid UUID
```

Comment:
This needs to be reverted as the validation script will be generated as part of CCM-1655

Reply:
done